config/spinnaker.yml

```diff
@@ -191,6 +191,7 @@ providers:
       # spinnaker installation runs as user "spinnaker" whose $HOME is
       # /home/spinnaker). The primaryCredentials.name will identify which profile
       # to use (for .aws/credentials).
+    defaultKeyPairTemplate: "{{name}}-keypair"
```

Comment:
Probably worth a comment here to explain the context within which the template will be expanded. A reader may be curious about the `{{name}}` bit without some explanation. Please put the same comment here and in `default-spinnaker-local.yml`.